WC_Layers/TwoLayers_Geometry.py

```diff
@@ -0,0 +1,21 @@
+import openmc
+
+def make_spherical_shells(layers, inner_radius, outer_boundary_type):    

```

Comment:
very minor point: for me it's more logical to start with the `inner_radius` because then your function arguments come in the same order as the geometry.